src/class/gc.rs

```diff
@@ -15,13 +15,8 @@ pub struct PyTraverseError(c_int);
 /// GC support
 #[allow(unused_variables)]
 pub trait PyGCProtocol<'p>: PyTypeInfo {
-    fn __traverse__(&'p self, visit: PyVisit) -> Result<(), PyTraverseError> {
-        unimplemented!()
-    }
-
-    fn __clear__(&'p mut self) {
-        unimplemented!()
-    }
+    fn __traverse__(&'p self, visit: PyVisit) -> Result<(), PyTraverseError>;
```

Comment:
Let's rename `visit` to `_visit` and remove `#[allow(unused_variables)]`

Reply:
I don't think it needs to be renamed, since there's no implementation.

Reply:
I removed the `#[allow(unused_variables)]`. Clippy doesn't complain about `visit`.